### ISSUE TITLE
More info dialogue triggered from Browser_mod pop up showing behind the browser mod pop up

### ISSUE THREAD

**Issue by igiannakas** [thomasloven/hass-browser_mod#603]
### Current Behavior

With mushroom cards v3.03 when a more info dialogue triggered from a mushroom entity card residing in a browser_mod pop up, it is showing it behind the browser mod pop up.

![Screenshot 2023-07-14 at 08 47 26](https://github.com/piitaya/lovelace-mushroom/assets/59056762/136a8c2b-79dc-4cf9-b88b-96f2b9cf899f)

![Screenshot 2023-07-14 at 08 47 46](https://github.com/piitaya/lovelace-mushroom/assets/59056762/5037fdf4-5840-49f7-a397-dfba59eec929)

In version 2.8.1 it works correctly. This is related to the change done due to the move to hass-action

![Screenshot 2023-07-14 at 08 54 09](https://github.com/piitaya/lovelace-mushroom/assets/59056762/fb642643-4928-4048-9291-151bd665c1a1)

The latest revision of browser mod is used, which is fixing the dialogue not showing in the first place. (#598 )

### Expected Behavior

The more info dialogue is shown infront of the browser mod pop up or the browser mod pop up is dismissed and the more info dialogue is showing.

### Steps To Reproduce

Version 3.03 mushroom cards, latest version of browser mod addressing defect 598 is installed
A browser mod pop up is created with a mushroom card in it
The mushroom card is linked to show the more info dialogue
Once clicked the more info dialogue is showing but behind the current pop up.

### Environment

```markdown
- Browser:Safari 16.5. Also on IOS 16.5.1
- HA Version: 2023.7.2
- Mushroom: 3.03
- Browser mod - latest revision addressing #598  
```

**Comment by thomasloven** [comment]
Fixed by #605

**Comment by igiannakas** [comment]
Confirming, works great thank you
